flexget/plugins/list/ombi_list.py

```diff
@@ -0,0 +1,540 @@
+            "api/v1/Request/movie/available"
+            if self.entry_type == 'movie'
+            else "api/v1/Request/tv/available"
+        )
+
+        data = {"id": self.data["requestId"]}
+
+        response = self._post_data(api_endpoint, data)
+
+        if not response:
+            # Unable to mark this item as requested, but what now?
+            return
+
+        if response.get('isError'):
+            log.error(
+                f"Failed to mark {self.data['title']} as available because: {response['errorMessage']}"
+            )
+
+        log.verbose(f"{self.data['title']} has been marked available.")
+
+
+class OmbiMovie(OmbiEntry):
+    entry_type = 'movie'
+
+    def __init__(
+        self, base_url: str, auth: Callable[[], Dict[str, str]], data: Dict[str, Any]
+    ) -> None:
+        super().__init__(base_url, auth)
+        self.data = data
+
+    @staticmethod
+    def from_imdb_id(base_url: str, tmdb_id: str, auth: Callable[[], Dict[str, str]]):
+
+        headers = {'Content-Type': 'application/json', 'Accept': 'application/json'}
+        headers.update(auth())
+
+        url = f"{base_url}/api/v2/Search/{OmbiMovie.entry_type}/{tmdb_id}"
+
+        response = requests.get(url, headers=headers)
+
+        if not response.ok:
+            log.error(f"{url} return {response.reason}")
+            # Do I throw an exception here? Is that allowed from a plugin?
+            return None
+
+        data = response.json()
+
+        # Their is a bug in OMBI where if you get a movie by its TMDB ID
+        # then the theMovieDbId field will be blank for some reason...
+        data['theMovieDbId'] = tmdb_id
+
+        return OmbiMovie(base_url, auth, data)
+
+
+class OmbiTv(OmbiEntry):
+    entry_type = 'tv'
+
+    def __init__(
+        self, base_url: str, auth: Callable[[], Dict[str, str]], data: Dict[str, Any]
+    ) -> None:
+        super().__init__(base_url, auth)
+        self.data = data
+
+    @staticmethod
+    def from_tvdb_id(base_url: str, tmdb_id: str, auth: Callable[[], Dict[str, str]]):
+
+        headers = {'Content-Type': 'application/json', 'Accept': 'application/json'}
+        headers.update(auth())
+
+        url = f"{base_url}/api/v2/Search/{OmbiTv.entry_type}/moviedb/{tmdb_id}"
+
+        response = requests.get(url, headers=headers)
+
+        if not response.ok:
+            log.error(f"{url} return {response.reason}")
+            # Do I throw an exception here? Is that allowed from a plugin?
+            return None
+
+        data = response.json()
+
+        # Their is a bug in OMBI where if you get a movie by its TMDB ID
+        # then the theMovieDbId field will be blank for some reason...
+        data['theMovieDbId'] = tmdb_id
+
+        return OmbiTv(base_url, auth, data)
+
+
+class OmbiSet(MutableSet):
+    supported_ids = ['imdb_id', 'tmdb_id', 'ombi_id']
+    schema = {
+        'type': 'object',
+        'properties': {
+            'base_url': {'type': 'string'},
```

Comment:
Very few plugins use "base_url" and "port" in configuration. 

If the connection is just http/https we should just use "url" consistently everywhere.

There are few plugins that we should probably consolidate into this format ....

@gazpachoking opinions?

Reply:
So use url like `url: https://127.0.0.1:5050`? I'm okay with that.

Reply:
Okay, will fix on my next commit.

Reply:
Fixed